devices/sonoff/snzb-03p_motion_sensor.json

```diff
@@ -0,0 +1,192 @@
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/battery",
+          "awake": true,
+          "refresh.interval": 7265,
+          "read": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 1,
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 1,
+            "eval": "Item.val = Attr.val / 2;",
+            "fn": "zcl:attr"
+          },
+          "default": 0
+        },
+        {
+          "name": "config/delay",
+          "refresh.interval": 3600,
+          "read": {
+            "at": "0x0020",
+            "cl": "0x0406",
+            "ep": 1,
+            "fn": "zcl:attr"
+          },
+          "write": {
+            "at": "0x0020",
+            "cl": "0x0406",
+            "dt": "0x21",
+            "ep": 1,
+            "eval": "Item.val",
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x0020",
+            "cl": "0x0406",
+            "ep": 1,
+            "eval": "Item.val = Attr.val",
+            "fn": "zcl:attr"
+          },
+          "default": 60
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/dark",
+          "read": {
+            "fn": "none"
+          },
+          "parse": {
+            "at": "0x2001",
+            "cl": "0xfc11",
+            "ep": 1,
+            "eval": "if (Attr.val === 0) { Item.val = true } else { Item.val = false }",
+            "fn": "zcl:attr"
+          },
+          "default": false
+        },
+        {
+          "name": "state/lastupdated"
+        },
+        {
+          "name": "state/presence",
+          "read": {
+            "fn": "none"
+          },
+          "parse": {
+            "at": "0x0000",
+            "cl": "0x0406",
+            "ep": 1,
+            "eval": "Item.val = Attr.val",
+            "fn": "zcl:attr"
+          },
+          "default": false
+        }
+      ]
+    }
+  ],
+  "bindings": [
+    {
+      "bind": "unicast",
+      "src.ep": 1,
+      "cl": "0x0406",
+      "report": [
+        {
+          "at": "0x0000",
+          "dt": "0x18",
+          "min": 1,
+          "max": 300
+        },
+        {
+          "at": "0x0020",
+          "dt": "0x21",
+          "min": 5,
+          "max": 60
+        }
+      ]
+    },
+    {
+      "bind": "unicast",
+      "src.ep": 1,
+      "cl": "0xFC11",
+      "report": [
+        {
+          "at": "0x2001",
+          "dt": "0x20",
+          "min": 1,
+          "max": 300,
+          "change": "0x00000001"
+        }
+      ]
+    },
+    {
+      "bind": "unicast",
+      "src.ep": 1,
+      "cl": "0x0001",
+      "report": [
+        {
+          "at": "0x0021",
+          "dt": "0x20",
+			"min": 3600,
+			"max": 7200,
+			"change": "0x00000002"
```

Comment:
You might want to ament the indentation.

Reply:
corrected